crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -87,6 +97,12 @@ impl Epoch {
         )))
     }
 
+    /// The total number of transaction blocks in this epoch.
+    async fn total_transactions(&self) -> Result<Option<BigInt>> {
```

Comment:
We should use `u64` for this, not `BigInt`, because we know it won't exceed `2^53`.